checker_test.cpp

```diff
@@ -0,0 +1,18 @@
+#include <assert.h>
+#include <iostream>
+#include "checker.hpp"
+#include <string>
+using namespace std;
+
+int main() {
+	assert(tempOfBatteryIsOk(25) == true);
```

Comment:
Test cases need to be strengthened further. Permutation and combinations of all three parameter values should be considered for batteryIsOk(..). For example one parameter will have 2 boundary values and 3 in between values, i.e. <, ==, > threshold values.